README.md

```diff
@@ -1,8 +1,8 @@
 # hubs-rock-kit
 
-This repository contains some organic rock assets for building scenes in in [Spoke](https://hubs.mozilla.com/spoke). In this repo, you will find the Blender files, as well as textures and the deployment script that is used to publish to Spoke. 
+This repository contains some organic rock assets for building scenes in in [Spoke]. In this repo, you will find the Blender files, as well as textures and the deployment script that is used to publish to Spoke. 
```

Comment:
Yvonne has been linking to the GitHub repository for Spoke, which I think is a good idea, so how about this:
```suggestion
This repository contains some organic rock assets for building scenes in in [Spoke](https://github.com/Hubs-Foundation/Spoke). In this repo, you will find the Blender files, as well as textures and the deployment script that is used to publish to Spoke. 
```